src/bin-utils/initialize/stringify-object.js

```diff
@@ -0,0 +1,25 @@
+import {isPlainObject} from 'lodash'
+
+function stringifyObject(object, indent) {
```

Comment:
Could we add a comment on this? I'm starting to forget what all this code is supposed to do 🙃